modules/apps/frontend-taglib/frontend-taglib-clay-sample-web/src/main/resources/META-INF/resources/js/holi.js

```diff
@@ -0,0 +1,3 @@
+export default function holi() {
```

Comment:
This would be a regular browser module.

Note that this is put inside the JAR verbatim: we don't need to build anything.